python/GafferUI/NumericSlider.py

```diff
@@ -128,12 +129,29 @@ def valueChangedSignal( self ) :
 
 		return self.__valueChangedSignal
 
+	def _snapPosition( self, position ) :
+
+		value = self._valueForPosition( position )
+		return self._positionForValue( self._snapValue( value ) )
+
+	def _snapValue( self, value ) :
```

Comment:
Just in the sense that `_snapPosition` is protected such that derived classes such as `NumericSlider` can re-implement snapping to their value space, `_snapValue` would allow classes derived from `NumericSlider` to affect their own value snapping if desired.  If we want to be really conservative with what we expose until we have an actual concrete use case, it can totally be private.